tests/common/reboot.py

```diff
@@ -319,7 +319,14 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
     logger.info('DUT {} create a file /dev/shm/test_reboot before rebooting'.format(hostname))
     duthost.command('sudo touch /dev/shm/test_reboot')
     # Get reboot-cause history before reboot
-    prev_reboot_cause_history = duthost.show_and_parse("show reboot-cause history")
```

Comment:
`prev_reboot_cause_history ` is used only for T2, so add the T2 check here to avoid initializing a variable that never gets used, and call CLIs that never are used. 

`if duthost.get_facts().get("modular_chassis") and reboot_type == REBOOT_TYPE_POWEROFF:`